Modules/System/Azure Blob Services API/src/Helper/ABSFormatHelper.Codeunit.al

```diff
@@ -111,6 +111,40 @@ codeunit 9044 "ABS Format Helper"
         exit(Document);
     end;
 
+    [NonDebuggable]
+    procedure XmlDocumentToTagsDictionary(Document: XmlDocument): Dictionary of [Text, Text]
+    var
+        Tags: Dictionary of [Text, Text];
+        TagNodesList: XmlNodeList;
+        TagNode: XmlNode;
+        KeyValue: Text;
+        Value: Text;
+    begin
+        if not Document.SelectNodes('/Tags/TagSet/Tag', TagNodesList) then exit;
```

Comment:
This one-liner should be expanded in to two lines ;) Maybe even an empty line after that.

Reply:
Fixed